packages/shared/src/utils/files/file-uploader-utils.js

```diff
@@ -41,7 +40,7 @@ export const readFiles = files => {
 
             fr.onerror = () => {
                 resolve({
-                    message: localize('Unable to read file {{name}}', { name: f.name }),
+                    message: `Unable to read file ${f.name}`,
```

Comment:
This is mostly an internal file handler error, so we don't really need `localize` for it.